<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QWidget" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>纠正相机:</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>181</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>曝光:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_adjust_exposure"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>增益:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>OffsetX:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>OffsetY:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Width:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Height:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_adjust_gain"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_adjust_offsetX"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="le_adjust_offsetY"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="le_adjust_width"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="le_adjust_height"/>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="btn_adjust_save">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>201</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>看缝相机:</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>181</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>曝光:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>增益:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>OffsetX:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>OffsetY:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Width:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Height:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_weld_exposure"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_weld_gain"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_weld_offsetX"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="le_weld_offsetY"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="le_weld_width"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="le_weld_height"/>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="btn_weld_save">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>201</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>识别针尖相机:</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>181</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>曝光:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>增益:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>OffsetX:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>OffsetY:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>Width:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Height:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_needle_exposure"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_needle_gain"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_needle_offsetX"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="le_needle_offsetY"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="le_needle_width"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="le_needle_height"/>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="btn_needle_save">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>430</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>配置参数更新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
